core/src/test/scala/unit/kafka/server/CreateTopicsRequestWithPolicyTest.scala

```diff
@@ -55,11 +63,9 @@ class CreateTopicsRequestWithPolicyTest extends AbstractCreateTopicsRequestTest
       assignment = Map(0 -> List(1, 0), 1 -> List(0, 1))))))
   }
 
-  @Test
-  def testErrorCreateTopicsRequests(): Unit = {
-    val existingTopic = "existing-topic"
-    createTopic(existingTopic, 1, 1)
```

Comment:
We don't support creating topics directly by writing metadata-topic, so we can't test this case in kraft mode.

Reply:
Do we need to? Seems like the main thing we're testing here is the case when the topic already exists. Seems like it would be just as good to create with the admin client and then verify the `TOPIC_ALREADY_EXISTS` error code.

Reply:
Thank you for pointing out this, firstly I think we are testing creating a topic that violates the policy.